src/tests/work-item/work-item-list/quickadd-workitem.spec.js

```diff
@@ -39,18 +38,17 @@ describe('Work item list', function () {
   });
 
   it('Creating a new quick add work item and Cancel delete - phone.', function () {
-    setBrowserMode('phone');
```

Comment:
it looks like you removed this line about setting the browser mode, don't you need this?